src/tests/Microsoft.PowerFx.Core.Tests.Shared/AnalyzeThreadSafety.cs

```diff
@@ -19,6 +18,9 @@ namespace Microsoft.PowerFx.Core.Tests
     /// </summary>
     public class AnalyzeThreadSafety
     {
+        // We cannot use this code with .Net 5.0+ as it uses System.Runtime.CompilerServices.IsExternalInit hack defined in PFX.Core and which is only valid pre-Net 5.0
+        // The symbol would be defined twice
```

Comment:
fwiw - this is probably fine since this is static analysis that should yield same results regardless of fx version

Reply:
I agree but would have preferred these tests to run on .Net 7 instead